launch/driver_params.yaml

```diff
@@ -1,4 +0,0 @@
-pacmod3_driver:
-  ros__parameters:
-    vehicle_type: "POLARIS_GEM"
-    frame_id: "pacmod"
```

Comment:
What does this `frame_id` of `pacmod` really mean?

Reply:
Just the frame_id used in the published message headers.